mlflow/pyfunc/backend.py

```diff
@@ -31,13 +31,18 @@ def __init__(self, config, workers=1, no_conda=False, install_mlflow=False, **kw
         self._no_conda = no_conda
         self._install_mlflow = install_mlflow
 
-    def prepare_env(self, model_uri):
+    def prepare_env(self, model_uri, capture_output=False):
         local_path = _download_artifact_from_uri(model_uri)
         if self._no_conda or ENV not in self._config:
             return 0
         conda_env_path = os.path.join(local_path, self._config[ENV])
         command = 'python -c ""'
-        return _execute_in_conda_env(conda_env_path, command, self._install_mlflow)
+        return _execute_in_conda_env(
```

Comment:
On second thought, can we split `_execute_in_conda_env` into two functions, one that creates an env and returns the env name, one that takes an env name and a command to run?

Reply:
We already have the function `get_or_create_conda_env`
but "prepare_env" still calls `_execute_in_conda_env` I think it is for running `python -c ""` immediately after conda env created in order to validate the env works ?

Reply:
Can we pull `get_or_create_conda_env` out of `_execute_in_conda_env`?